src/diffusers/models/unet_2d_blocks.py

```diff
@@ -1945,6 +1955,30 @@ def custom_forward(*inputs):
         return hidden_states
 
 
+class MOVQAttention(nn.Module):
```

Comment:
I think we don't need to create a new class anymore now this PR is merged https://github.com/huggingface/diffusers/pull/3387